README.md

```diff
@@ -48,7 +48,7 @@ to have production-ready php settings.
  * Edit the `.env` file located in the `rdc` folder review the contents, making changes as necessary.
  * Once `.env` file settings are correct, from the `rdc` folder type `docker-compose up -d`
  * Open your web browser and goto `http://localhost` (or, in some cases with macs `http://127.0.0.1`) and follow
-  directions for further installation
+  directions for further installation -> https://docs.google.com/document/d/1UsQmK1Jw0L_wD3UcjlQo_6iDjXadLeFHT427e50XR8M/edit
```

Comment:
I don't have access to this Google Doc. You could make it public, but moving the content into the documentation in this repo is probably a better idea. That makes it easier for the documentation to evolve at the same time the features of the installer evolve. If the docs are separate from the code, it's harder to keep them in synchrony.